app/controllers/api/v0/api_controller.rb

```diff
@@ -149,10 +159,17 @@ def incidents_search
   end
 
   def omni_search
-    # We intentionally exclude Post, as our autocomplete ui isn't very useful with
-    # them yet.
-    params[:persons_table] = true
-    search(Competition, User, Regulation, Incident)
+    model_names = params[:models]
+    if model_names.present?
+      model_list = model_names.split(",")
+      models = model_list.map { |model_name| Api::V0::ApiController.search_models[model_name] }
+      search(*models)
+    else
```

Comment:
I think this PR has been re-purposed and multiple models are not supported anymore.
So probably this controller handling can go away.

Reply:
Done